particles/List/source/Items.js

```diff
@@ -66,25 +61,33 @@ defineParticle(({DomParticle, resolver, html}) => {
     shouldRender({list}) {
       return Boolean(list);
     }
+    getId(entity) {
+      // TODO(sjmiles): ensure we get the entity.id even if the rawData has it's own id,
+      // discuss with Runtime folks (either we disallow `id` collision or have a more obvious way to get this)
+      return Object.getOwnPropertyDescriptor(entity.__proto__.__proto__, 'id').get.apply(entity);
+    }
     render({list, selected}) {
       const selectedId = selected && selected.id;
+      const sorted = list.sort((a, b) => a.name > b.name ? 1 : a.name === b.name ? 0 : -1);
```

Comment:
use localeCompare?

Reply:
This is a good catch, however `localeCompare` is really super slow. There are ways to make it faster, but they are involved. I think it's best to wait until we get around to tackling localisation more broadly (which should be early in the process IMO).